File: Python/41-51/e_47.py
# Escribe una función en python minimo(L) que tome una lista de enteros L como parámetro y devuelva el valor más pequeño.


def minimo(L):
    if not L:
        # Manejar el caso de una lista vacía
        return None
    
    # Utilizar la función min para obtener el valor mínimo de la lista
    return min(L)

# Ejemplo de uso
lista_numeros = [10, 5, 8, 15, 3]
resultado = minimo(lista_numeros)
print(resultado)



# Esta función utiliza la función incorporada min de Python para encontrar el valor mínimo en la lista de enteros proporcionada.
# Si la lista está vacía, la función devuelve None. Puedes probar la función con diferentes listas de enteros para encontrar el valor mínimo.

File: JS/41-50/e_47.js
//  Escribe una función en python minimo(L) que tome una lista de enteros L como parámetro y devuelva el valor más pequeño.

function minimo(L) {
    if (L.length === 0) {
        // Manejar el caso de un array vacío
        return null;
    }

    // Utilizar Math.min para obtener el valor mínimo del array
    return Math.min(...L);
}

// Ejemplo de uso
const arrayNumeros = [10, 5, 8, 15, 3];
const resultado = minimo(arrayNumeros);
console.log(resultado);


// En este ejemplo, la función minimo utiliza el operador de propagación (...) para pasar los elementos del array como argumentos individuales a la función Math.min.
// Si el array está vacío, la función devuelve null. Puedes probar la función con diferentes arrays de enteros para encontrar el valor mínimo.